enterprise_access/apps/api/v1/views.py

```diff
@@ -134,6 +159,119 @@ def _validate_subsidy_request(self):
             logger.exception(error_msg)
             raise SubsidyRequestCreationError(error_msg, status.HTTP_422_UNPROCESSABLE_ENTITY)
 
+    def _validate_subscription_plan_uuid(self, subscription_plan_uuid):
+        """
+        Raises SubsidyRequestError if subsidy_request_uuid not present or invalid
```

Comment:
subsidy_request_uuid  -> subscription_plan_uuid in comments here

Reply:
ty